### DISCUSSION TITLE
Please, I need help to Install Without Docker

### DISCUSSION THREAD

**Discussion by zouzeTG** [open-webui/open-webui#1393] [Q&A]
Hi everyone, 
Thank's for this great tool ! 
I have followed the procedure to install and use open-webui without docker, for development purposes : https://docs.openwebui.com/getting-started
But I get errors like : 
<3>WSL (9) ERROR: CreateProcessEntryCommon:505: execvpe /bin/bash failed 2
<3>WSL (9) ERROR: CreateProcessEntryCommon:508: Create process not expected to return

or  ERROR:    Error loading ASGI app. Could not import module "main".

WSL is installed and I use VSCode in windows server environment

Thank's  a lot for your help

**Comment by BetterthanNothing420** [comment]
use pinokio

**Reply by justinh-rahb** [reply]
Pinokio uses Python 3.12 which our project is not compatible with officially. Your mileage may vary.
